frontend/src/components/profile-bids-tab/ProfileBidsTab.jsx

```diff
@@ -0,0 +1,65 @@
+import React, {useEffect, useState} from "react";
+import "./ProfileBidsTab.css";
+import Button from "components/button/Button";
+import {getBidsForUser} from "api/Bid";
+import { getDateDiffernece } from "utils/DateHelper";
+import moment from "moment";
+import { Link } from "react-router-dom";
+import { faGavel } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+function ProfileBidsTab() {
+  const [bids, setBids] = useState([]);
+
+  useEffect(() => {
+    const getBids = async () => {
+      const res = await getBidsForUser();
+      setBids(res);
+    };
+    getBids();
+  }, []);
+
+  return (
+    <div className="profile-bids-table">
+      <div className="gray-heading">
+        <div className="bids-table-item">Item</div>
+        <div className="bids-table-name">Name</div>
+        <div className="bids-table-left">Time Left</div>
+        <div className="bids-table-price">Your Price</div>
+        <div className="bids-table-bids">No. Bids</div>
+        <div className="bids-table-highest">Highest Bid</div>
+        <div className="bids-table-empty"></div>
+      </div>
+      {bids.length !== 0 ? bids.map((bid, index) => (
+        <div className="bids-table-row" key={index}>
```

Comment:
Using index as key can lead to subtle bugs, read more [here](https://react.dev/learn/rendering-lists#why-does-react-need-keys). If there is no ordering, inserting or deleting items in the table, it's fine.